pkgs/by-name/lo/loco-cli/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, rustPlatform, fetchFromGitHub, stdenv }:
+rustPlatform.buildRustPackage rec {
+  pname = "loco-cli";
+  version = "0.2.6";
+
+  src = fetchFromGitHub {
+    owner = "loco-rs";
+    repo = "loco";
+    rev = "51e0362";
+    hash = "sha256-ZiAl+Ru2ggLy7RRqQySwKRbWtGesR7ZgREIpHKrJ00Q=";
+    sparseCheckout = [ "loco-cli" ];
+  };
+
+  cargoLock = { lockFile = ./Cargo.lock; };
+
+  postPatch = ''
+    ln -s ${./Cargo.lock} Cargo.lock
+  '';
+
+  sourceRoot = "source/loco-cli";
+
+  cargoSha256 = lib.fakeSha256;
```

Comment:
Please replace this with the actual hash!

Reply:
Removed as its not needed with `cargoLock.lockFile`. (see https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/rust.section.md#importing-a-cargolock-file-importing-a-cargolock-file)